pkg/kp/healthmanager.go

```diff
@@ -208,6 +208,9 @@ func processHealthUpdater(
 			if ok {
 				logger.NoFields().Debug("new health status: ", h.Status)
 				localHealth = &h
+				if !healthEquiv(localHealth, remoteHealth) {
```

Comment:
Is your suggestion to add another health variable to this function, a la `previousLocalHealth` and compare to that?